datanymizer_engine/Cargo.toml

```diff
@@ -8,16 +8,16 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1.0"
-chrono = "0.4"
 config = "0.10"
 csv = "1.1"
-fake = { version = "2.4.1", features = ["chrono"] }
+fake = { version = "2.4.1" }
 once_cell = "1.5.2"
 rand = "0.8.4"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_yaml = "0.8.14"
-tera = "1.15.0"
+tera = { version = "1.15.0", default-features = false, features = ["slug", "percent-encoding", "humansize", "rand"] }
```

Comment:
This change breaks one of my templates that randomly generates a string because "get_random" is a "builtin" feature and this is excluding it.  See https://tera.netlify.app/docs/#get-random

Snippet from my config.yaml
```
format: '{% set r = get_random(start=1,end=9) %}{% if r == 1 %} ...
```

Fix is to add "builtins" to the features list:
tera = { version = "1.15.0", default-features = false, features = ["slug", "percent-encoding", "humansize", "rand", "builtins"] }

Reply:
You can use [this branch](https://github.com/datanymizer/datanymizer/tree/fix_get_random) as a quick solution...

Reply:
After looking I see that `chrono` is not well maintained and depends on an old version of the `time` library with a known segv vulnerability.  Is the `Tera` project tracking this and perhaps working on a way to fix it?

Reply:
As far as I can see, in Tera it is necessary just to accurately separate features by flags.

Reply:
I just finished the tera changes to put all the defn and use of chrono things under the existing "chrono" feature, and updated the defaults.

https://github.com/Keats/tera/pull/711

Reply:
Thank you. It would be great if this change is merged.

Reply:
@evgeniy-r As a temporary and quick solution, we can use the fork of @mbeynon with `chrono` isolation until it is accepted to upstream.